datanode/gateway/graphql/schema.graphql

```diff
@@ -542,6 +542,32 @@ type Query {
     withdrawalId: ID!
   ): Erc20WithdrawalApproval
 
+  erc20MultiSigSignerAddedBundles(
+    "The node id of the validator of which a signature bundle is required"
+    nodeId: ID!
+    "The epoch which generated the bundle i.e the epoch in which the node was demoted from a tendermint validator"
+    epochSeq: String!
+    "Pagination"
+    pagination: Pagination
+  ): ERC20MultiSigSignerAddedConnection!
+
+  erc20MultiSigSignerRemovedBundles(
+    "The node id of the validator of which a signature bundle is required"
+    nodeId: ID!
+    "The ethereum address of the validator which will submit the bundle"
+    submitter: String!
+    "The epoch which generated the bundle i.e the epoch in which the node was demoted from a tendermint validator"
+    epochSeq: String!
```

Comment:
The ! means non-nullable right? I think we definitely want `epochSeq` to nullable, and maybe even `submitter`.